src/config.rs

```diff
@@ -1712,6 +1715,17 @@ impl UnifiedReadPoolConfig {
                     .into(),
             );
         }
+        let limit = cmp::max(
+            UNIFIED_READPOOL_MIN_CONCURRENCY,
+            SysQuota::cpu_cores_quota() as usize,
+        );
+        if self.max_thread_count > limit {
```

Comment:
Since now `max_thread_count` means `core_thread_count` in yatp, it lower threshold should be min_threshold not `UNIFIED_READPOOL_MIN_CONCURRENCY`